jwql/tests/test_pipeline_tools.py

```diff
@@ -41,7 +47,7 @@ def test_completed_pipeline_steps():
         File to be checked
     """
 
-    filename = os.path.join(get_config()['filesystem'], 'jw00312',
+    filename = os.path.join(get_config()['proprietary_filesystem'], 'jw00312', 'jw00312002001',
```

Comment:
Are we sure this file will always be in the proprietary filesystem? Could it migrate to the public filesystem eventually?